.github/workflows/ci.yml

```diff
@@ -82,8 +82,8 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        python-version: ["3.8"]
-        nautobot-version: ["1.4.0"]
+        python-version: ["3.10"]
```

Comment:
why 3.10?

Reply:
I'm testing with this version on my local, updates to this are mentioned in `To Do`s in this PR.